### PR TITLE
Replace ORA XQueue vars

### PR DESCRIPTION
**Pull request by wedaly**
The ORA role depends on variables defined in XQueue, which aren't available when deploying ORA independently of XQueue.  This change replaces the XQueue-defined variables with ORA-specific equivalents.

I tested ORA was able to pick up submissions off the XQueue in a sandbox, which was not the case before.

@feanil

### CODE REVIEW

File: playbooks/roles/ora/defaults/main.yml [hunk 1]
```diff
@@ -54,6 +54,9 @@ ORA_USERS:
   "lms": "password"
 
 ORA_XQUEUE_URL: "http://localhost:18040"
+ORA_XQUEUE_DJANGO_USER: "lms"
+ORA_XQUEUE_DJANGO_PASSWORD: "password"
+ORA_XQUEUE_BASIC_AUTH: [ "edx", "edx" ]
```
Comment (feanil) [inline playbooks/roles/ora/defaults/main.yml]: Please make this two seperate variables BASIC_AUTH_USER and BASIC_AUTH_PASSWORD

**Comment by wedaly** [comment]
@feanil No problem; fixed

**Comment by jarv** [comment]
:+1:

**Comment by feanil** [comment]
looks good.

**Pull request merged**